birdhouse/optional-components/wps-healthchecks/docker-compose-extra.yml

```diff
@@ -0,0 +1,80 @@
+version: "3.4"
+services:
+  catalog:
+    healthcheck:
+      test:
+        [
+          "CMD",
+          "python",
+          "-c",
+          "import requests; \
+           assert requests.get('http://localhost?service=WPS&request=GetCapabilities').status_code == 200",
```

Comment:
This one on port 80 direct?

Reply:
Yes. Catalog defines port mapping `8086:80`. It's app is running on default HTTP port.